src/routes/acquirerRoutes.ts

```diff
@@ -123,15 +147,44 @@ router.get("/:resourceID/:step", async (req: Request, res: Response) => {
   const stepData = formdata.steps[formStep];
   const assetTitle = formdata.assetTitle;
 
+  if (req.query.action === "back" && formdata.stepHistory) {
+    formdata.stepHistory.pop();
+  }
+
   if (skipThisStep(formStep, formdata)) {
     stepData.skipped = true;
     return res.redirect(`/acquirer/${resourceID}/${stepData.nextStep}`);
   }
 
+  let backLink = null;
+
+  if (!formdata.stepHistory) {
+    formdata.stepHistory = [];
+  }
+
+  // If we have history and are pressing back, we set the back link.
+  if (formdata.stepHistory.length > 1) {
+    backLink = `/acquirer/${resourceID}/${
+      formdata.stepHistory[formdata.stepHistory.length - 2]
+    }?action=back`;
+  }
```

Comment:
This block can be deleted. 
There's an `if/else` below that always sets the backlink to something. 
So whatever you set the backLink to here is always going to get immediately overwritten by the bit below.
```suggestion
```